odoo/tools/misc.py

```diff
@@ -1559,6 +1559,8 @@ def format_time(
     locale = babel_locale_parse(lang.code)
     if not time_format:
```

Comment:
Maybe also use the Odoo's format for *medium*?
```suggestion
    if not time_format or time_format == 'medium':
```

Reply:
Yes, sure!
We can do for medium also.